modules/apps/powwow/powwow-portlet/docroot/WEB-INF/src/com/liferay/powwow/meetings/portlet/MeetingsPortlet.java

```diff
@@ -379,6 +396,57 @@ public void updatePowwowMeeting(
 		}
 	}
 
+	protected void checkMaxOccurrence(ResourceRequest resourceRequest, ResourceResponse resourceResponse)
+			throws IOException, InvalidRecurrenceRuleException, PortalException {
+		ServiceContext serviceContext = ServiceContextFactory.getInstance(
+				PowwowMeeting.class.getName(), resourceRequest);
+
+		DateTimeFormatter formatter = DateTimeFormat.forPattern("MM/dd/yyyy");
```

Comment:
i'm curious if we should have dynamic pattern base on the portal setting or user's setting?

Reply:
We need to format the date which return by LocalDateIterable